client/components/AdoptionForm.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from "react"
+import { Redirect } from "react-router-dom"
+import _ from "lodash"
+
+import ErrorList from "./ErrorList"
+
+const AdoptionForm = props => {
+  const [formData, setFormData] = useState({
+    name: "",
+    phoneNumber: "",
+    email: "",
+    homeStatus: "",
+    applicationStatus: "pending",
+    adoptablePetId: props.adoptablePetId
+  })
+  const [errors, setErrors] = useState({})
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+
+  const handleChange = event => {
+    setFormData({
+      ...formData,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+
+  const addNewAdoptionApplication = async () => {
+    try {
+      const response = await fetch("/api/v1/adoption-application", {
```

Comment:
conventionally/RESTfully, this should post to `/api/v1/adoption-applications` plural!